src/r_bsp.c

```diff
@@ -124,7 +124,7 @@ static void R_ClipWallSegment(int first, int last, boolean solid)
 
 void R_ClearClipSegs (void)
 {
-  memset(solidcol, 0, MAX_SCREENWIDTH);
+  memset(solidcol, 0, SCREENWIDTH << hires); // [Nugget] Dynamic arrays
```

Comment:
You should be able to get away with `viewwidth` whenever you use `SCREENWIDTH << hires` in this code change.